lib/thread.h

```diff
@@ -67,6 +68,7 @@ struct m0_thread_tls {
 	struct m0_thread_arch_tls  tls_arch;
 	struct m0_addb2_mach      *tls_addb2_mach;
 	struct m0_thread          *tls_self;
+	m0_processor_nr_t          tls_cpuid;
```

Comment:
[* suggestion *] Please make the name descriptive, assuming there may be more than 1 CPU core for a thread to execute on.

Reply:
Done.